test/model_factory_test.py

```diff
@@ -73,6 +85,12 @@ def setUp(self):
           model:
             cfg_type_name: a_type
             type: NamedModelElement
+        unused_type:
+          src:
+          - file: unused_type_values.xxx
```

Comment:
is this attribute actually required if there are no type instances (I did not test)?
if not, this could be omitted (same goes for the file)